robot/web/web_c.go

```diff
@@ -106,10 +106,17 @@ func (svc *webService) addNewStreams(ctx context.Context) error {
 		return nil
 	}
 
-	newStream := func(name string) (gostream.Stream, bool, error) {
+	newStream := func(name string, isVideo bool) (gostream.Stream, bool, error) {
 		// Configure new stream
-		config := *svc.opts.streamConfig
-		config.Name = name
+		config := gostream.StreamConfig{
+			Name: name,
+		}
+
+		if isVideo {
```

Comment:
This code path gets executed when a robot reconfigures and finds a new camera. This was the "bug"/behavior difference that's being changed.